flake.nix

```diff
@@ -13,56 +13,76 @@
 
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable-small";
-    flake-utils.url = "github:numtide/flake-utils";
-
-    flake-compat = {
-      url = "github:edolstra/flake-compat";
+    monad-schedule = {
+      url = "github:turion/monad-schedule";
```

Comment:
I don't quite understand why we would want to pull in `master` from this repo. My maintenance commitment is only to the Hackage published versions within the bounds in `rhine.cabal`.